### DISCUSSION TITLE
Help setting up development cluster.

### DISCUSSION THREAD

**Discussion by maxenko** [ravendb/ravendb#14213] [Q&A] [answered]
First time user here.

Ubuntu 22.04, RavenDB 5.3.

I am following this as example: https://www.youtube.com/watch?v=jBuilL6ColY

Problem is that my node installations immediately go to active status (without any databases) and I cannot add them to the cluster as that requires a passive node.

What could be the problem?

Further:
How can I remove a node from its own cluster? I cannot find anything in UI and not finding anything in documentation.

**Update 1:**
Was able to add a newly set up node to the first one... 3rd one is refusing, reinstalling it does not reset it back to pre-install, its already joined to a cluster after insecure setup.

Is there step-by-step documentation regarding this anywhere?

**Update 2:**

Until someone points me to a button that resets/removes the node from its own cluster, you can:

1. `apt remove ravendb`
2. `apt purge ravendb` <- this removes config files that stick around and block fresh re-install
3. Go through the wizard (in my case unsecure as its running on a local dev instances), and right after assigning IP, press restart... 
4. Go to your other active node and add this instance by IP.

I hope it gets better from here. :worried:

**Comment by georgiosd** [comment, marked as answer]
I think you'll have some better luck with this one: https://ravendb.net/docs/article-page/5.3/csharp/start/installation/manual

**Reply by maxenko** [reply]
Got it, thank you. Bypass it altogether. Will try next setup.

**Reply by georgiosd** [reply]
@maxenko I didn't watch the video, but it may be a little outdated, it's for 4.0. The wizard will guide you to setup a cluster.

Essentially what you need to do is:
a Run Raven on the first node and connect on the random port it gives you
b Add the certificate, other nodes and their hostnames
c Restart Raven
d Copy the files given to you after adding the certificate to nodes B+C

**Comment by ayende** [comment]
What was likely the issue is that you added the new node from the new node? You need to do everything from node A
